paper-api/src/main/java/org/bukkit/OfflinePlayer.java

```diff
@@ -290,13 +291,28 @@ public default BanEntry banPlayer(@Nullable String reason, @Nullable java.util.D
     // Paper end
 
     /**
-     * Gets the Location where the player will spawn at, null if they
+     * Gets the Location where the player will spawn at, {@code null} if they
+     * don't have a valid respawn point.
+     * <br>
+     * Unlike online players, the location if found will not be loaded by default.
```

Comment:
This note will be inherited by the player class which may seem confusing, we may want to have some javadocs in the player class override for this method still.